aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -62,7 +49,7 @@ proposals:
         enabled:
           - ed25519_pubkey_validate_return_false_wrong_length
         disabled: []
-  - name: step_5
+  - name: step_4
```

Comment:
@runtian-zhou maybe we should just name these as to what they are doing instead of indices since indices need to be manually updated